index.d.ts

```diff
@@ -0,0 +1,83 @@
+import { ServiceSchema, Context } from "moleculer";
+import { CronJob as NodeCronJob, CronTime as NodeCronTime } from "cron";
+
+declare module "moleculer" {
+    interface ServiceSchema {
+        cronJobs?: CronJobConfig[];
+    }
+}
+
+export interface CronJobConfig {
+    name: string;
+    cronTime: string | Date;
+    onTick: (this: Context) => void | Promise<void>;
+    onComplete?: (this: Context) => void;
+    start?: boolean;
+    timeZone?: string;
+    context?: any;
+    runOnInit?: boolean | ((this: Context) => void);
+    utcOffset?: number;
+    unrefTimeout?: boolean;
+}
+
+export interface CronJob {
+    start: () => void;
+    stop: () => void;
+    lastDate: () => Date;
```

Comment:
I think you're missing `setTime` here

```suggestion
    setTime: (time: CronTime) => void;
    lastDate: () => Date;
```